### DISCUSSION TITLE
Pass arguments during Loco project initialization

### DISCUSSION THREAD

**Discussion by dinosath** [loco-rs/loco#438] [Q&A] [answered]
Is possible to pass arguments in loco new and generate the project with essentialy one command, instead of running loco new and interacting with the promts ?

**Comment by kaplanelad** [comment, marked as answer]
yes, see the following example

LOCO_APP_NAME=saas LOCO_TEMPLATE=saas loco new

**Reply by dinosath** [reply]
That worked. Thanks for the reply!!!! Is this in the documentation and i missed it?

**Reply by kaplanelad** [reply]
This is not documented. you are more then welcome to open a pull request for that
